client/implementation/src/main/java/io/smallrye/graphql/client/impl/typesafe/reflection/MethodInvocation.java

```diff
@@ -262,18 +266,25 @@ public String getOperationTypeAsString() {
         }
     }
 
-    public String getGroupName() {
-        return groupName;
+    public List<String> getNamespaces() {
+        return namespaces;
     }
 
-    private String readGroupName(Method method) {
-        Name annotation = method.getDeclaringClass().getAnnotation(Name.class);
+    private List<String> readNamespace(Method method) {
+        Namespace annotation = method.getDeclaringClass().getAnnotation(Namespace.class);
         if (annotation != null) {
-            String groupName = annotation.value().trim();
-            if (!groupName.isEmpty()) {
-                return groupName;
-            }
+            return Arrays.stream(annotation.value().split("/"))
+                    .map(String::trim)
+                    .collect(Collectors.toList());
         }
-        return null;
+        return List.of();
+    }
+
+    public String getOperationName() {
+        return namespaces.stream().map(this::prettyString).collect(joining()) + prettyString(getName());
```

Comment:
For example
```
@Namespace("first/second/third/someMynotherName")
class Api {
   @Query("findAll")
   ////
}
```

If don create first letter to upper case, operation will firstsecondthirdsomeMynotherNamefindAll. it is unreadable.
And create it only findAll incorrect, becouse in code may be other findAll method. And if we will call just findAll, using some router (with analytics), it can break call analytics.

Reply:
Okay, I think I get it: if the user wants to send multiple operations in one request, their names have to be distinct. But with the typesafe client, you can send multiple request only with a `@Multiple` annotation. The operation name is only scoped within a query string, not globally. I think we can safely skip this. Try to write a test to prove me wrong 😜
BTW, this code _does_ uppercase the very first letter: `FirstSecondThirdSomeMynotherName` ... but the `F` could also remain `f` 😁

Reply:
When the Quarkus tests failed, I saw that Quarkus also checks the operation name. 
And maybe it’s worth making sure that the operation name is formed like this:
1. If there is `@Name` or `@Namespace`, capitalize the first letters. 
2. If not, leave it as is.

As a result, the quarkus build will not crash, and there will be a minimum changes in the tests.
As for `@Multiple`, I'll take a look today.

Reply:
I'm obviously biased by the Java naming convention that methods start with a lowercase character. That's why I'd like to stick with that, also for operation names. I.e., also if there is a `@Namespace` annotation, the very first character should be lowercase. E.g., `@Namespace("users") ... @Query List<User> all() {...` would result in an operation name `usersAll`.

Reply:
I just referred to examples from the graphql documentation https://graphql.org/learn/queries/#operation-name
But of course I can change it to your example, no problem.